pytket/pytket/qir/qir.py

```diff
@@ -19,11 +19,16 @@
 from pyqir.generator import SimpleModule, BasicQisBuilder, types  # type: ignore
 from pytket import Circuit, OpType, Bit, Qubit
 from pytket.circuit import Op  # type: ignore
+from pytket.circuit.logic_exp import BitWiseOp
 
 
 CustomPyQIRGate = NamedTuple(
     "CustomPyQIRGate",
     [
+        (
+            "optype",
```

Comment:
I'd suggest calling it something other than "optype" since that term is very overloaded in tket

Reply:
Definitely. I had the same reflection. Should be a bit more creative.

Reply:
In [07951c5](https://github.com/CQCL/tket/pull/332/commits/07951c562d645be16af5be0739bac9434fa6ee36)